examples-complex-import/authors/sqlc.yaml

```diff
@@ -0,0 +1,17 @@
+version: "1"
```

Comment:
The entire `examples-complex-import` directory should be moved to the `internal/endtoend/testdata` directory and renamed to `complex_go_type`.